File: Domains/ApplicationDomain/BOA/Entities/FeedHistoryDetail.cs
using ApplicationDomain.Core.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationDomain.BOA.Entities
{
    public class FeedHistoryDetail : EntityBase<int>
    {
        public string Code { get; set; }
        public int FeedHistoryId { get; set; }
        public FeedHistory FeedHistory { get; set; }
        public int FoodId { get; set; }
        public Food Food { get; set; }
        public int Quantity { get; set; }


    }
}

File: Applications/WebAdminApplication/ClientApp/src/app/modules/farmer/models/feedHistoryDetail.ts
export class FeedHistoryDetail {
  id: number;
  code: string;
  foodId: number;
  quantity: number;
  foodType: string;
  foodName: string;

  constructor() {
   this.id =0;
   this.code="";
   this.foodId=0;
   this.quantity = 0;
  }
}
